src/main/scripts/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
+    "skipLibCheck": true,
```

Comment:
If typescripts finds some kind of error in libs (`node_modules`) it won't throw any error. This is useful so the webpack build won't fail due to these errors (which we can't handle anyway)